programming-core/src/main/java/org/objectweb/proactive/core/httpserver/HTTPServer.java

```diff
@@ -122,12 +124,13 @@ public void stop() throws Exception {
         this.server.stop();
     }
 
-    /** destroy the HTTP server, terminates the http server thread 
+    /** destroy the HTTP server, terminates the http server threads
      * 
      * @throws Exception If the HTTP server fails to stop
      */
     public void destroy() throws Exception {
         this.server.destroy();
+        this.threadPool.terminate();
```

Comment:
Does it matter when the termination of the thread pool is done? Before destroying the server maybe?